views/cypress/tests/manage-schema.spec.js

```diff
@@ -148,6 +148,10 @@
             cy.renameSelectedNode(selectors.itemForm, selectors.editItemUrl, childItemName);
         });
 
+        it('appears error on save due to notEmpty restriction', function () {
+            cy.get('div[class="form-error"]').should('have.text', 'This field is required');
+        });
```

Comment:
nitpick: I would suggest to reword the test name so it reads as a English sentence ("it shows an error on save due to notEmpty restriction"). BTW I'm used to that kind of phrasing on tests, so maybe it is just a matter of personal preference :)

```suggestion
        it('shows an error on save due to notEmpty restriction', function () {
            cy.get('div[class="form-error"]').should('have.text', 'This field is required');
        });
```